### PR TITLE
address projects with incorrect labels

### LINKED ISSUE #1674: review projects with missing labels
Found these on a recent check of the labels:

```
Cannot find label 'hacktoberfest' for project 'OpenRCT2/OpenRCT2'
Cannot find label 'good first issue' for project 'raiden-network/raiden'
Cannot find label 'hacktoberfest' for project 'home-assistant/home-assistant.io'
```

This suggests:

 - a label is no longer used on the project
   - often this just means finding a suitable label to use in it's place
 - the label name has changed (either a typo or a full rename)
   - check the labels listed for the project, update our listing with the better match
 - the label has been dropped
   - probably a sign that a different label is now preferred
   - if no label looks suitable, this indicates the project is no longer interested in participating

### PR DESCRIPTION
**Pull request by XxZhang2017**
Resolves #1674 

Changing the label 'hacktoberfest' for project OpenRCT2/OpenRCT2 to 'good first issue', deleting file raiden.yml, changing the label 'hacktoberfest' for project home-assistant/home-assistant.io to 'help wanted'.

### CODE REVIEW

**Comment by XxZhang2017** [comment]
@shiftkey I am sorry that I didn't take the correct action for #1692 . 
I should have commit your changes. 
What should I do next to fix the issue?

File: _data/projects/OpenRCT2.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ tags:
 - rollercoasters
 - 2d
 upforgrabs:
-  name: hacktoberfest
+  name: good first issue
   link: https://github.com/OpenRCT2/OpenRCT2/labels/hacktoberfest
```
Comment (shiftkey) [inline _data/projects/OpenRCT2.yml]: This needs to be in sync with the label change

```suggestion
  link: https://github.com/OpenRCT2/OpenRCT2/labels/good%20first%20issue
```

File: _data/projects/home-assistant.io.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ tags:
 - hass
 - hassio
 upforgrabs:
-  name: hacktoberfest
+  name: help wanted
   link: https://github.com/home-assistant/home-assistant.io/labels/hacktoberfest
```
Comment (shiftkey) [inline _data/projects/home-assistant.io.yml]: This also needs to be in sync with the label change:

```suggestion
  link: https://github.com/home-assistant/home-assistant.io/labels/Help%20wanted
```

File: _data/projects/home-assistant.io.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@ tags:
 - hass
 - hassio
 upforgrabs:
-  name: hacktoberfest
+  name: help wanted
```
Comment (shiftkey) [inline _data/projects/home-assistant.io.yml]: This is case-sensitive:

```suggestion
  name: Help wanted
```

File: _data/projects/raiden.yml [hunk 4]
```diff
@@ -1,16 +0,0 @@
...
-site: https://github.com/raiden-network/raiden/
-tags:
-- blockchain
-- ethereum
-- payment-channels
-- microtransactions
-- state-channels
-upforgrabs:
-  name: good first issue
-  link: https://github.com/raiden-network/raiden/labels/good%20first%20issue
```
Comment (shiftkey) [inline _data/projects/raiden.yml:13]: I don't think we should delete this project - can you restore the file and update it to use this label:

https://github.com/raiden-network/raiden/labels/Tag%20%2F%20Good%20First%20Issue

**Review by shiftkey — CHANGES_REQUESTED** [review]

File: _data/projects/raiden.yml (hunk 4 above)
Comment (XxZhang2017) [reply _data/projects/raiden.yml:13]: I am so sorry about my mistake. I didn't check the second pages of the tags.
We changed the label `hacktoberfest` of project 'home-assistant/home-assistant.io' from  to `Help want`.
Should I change the link to 
https://github.com/home-assistant/home-assistant.io/labels/Help%20wanted

Comment (shiftkey) [reply _data/projects/raiden.yml:13]: > We changed the label hacktoberfest of project 'home-assistant/home-assistant.io' from to Help want.
Should I change the link to
https://github.com/home-assistant/home-assistant.io/labels/Help%20wanted

Yes, see my suggested change above: https://github.com/up-for-grabs/up-for-grabs.net/pull/1694/files#r346884336

File: _data/projects/raiden.yml [hunk 5]
```diff
@@ -10,7 +10,7 @@ tags:
 - state-channels
 upforgrabs:
   name: good first issue
```
Comment (shiftkey) [inline _data/projects/raiden.yml]: And this tag now needs to be updated:

```suggestion
  name: "Tag / Good First Issue"
```

**Review by shiftkey — APPROVED** [review]
Thanks for working through these changes for me @XxZhang2017!

**Pull request merged**